tsconfig.json

```diff
@@ -0,0 +1,8 @@
+{
+  "extends": "./packages/core/test/tsconfig.json",
+  "compilerOptions": {
+    "rootDir": "..",
+    "module": "Node16"
```

Comment:
Any specific reason to go for Node 16?

Reply:
That's from I tried to make sscce.ts an ESM module

Reply:
done